### DISCUSSION TITLE
Spontaneous deletion of data from all tables

### DISCUSSION THREAD

**Discussion by eduardvartanyan** [ClickHouse/ClickHouse#54432] [General]
Hello.

The database is located on Yandex Cloud. There are 120 tables in the database, which are automatically supplemented with data from the postgre database using PHP scripts.
Faced a problem that regularly all tables are cleared by themselves. There are no data clearing commands in scripts, only select and insert.

In one of the cases, when in test mode I filled the table with more than a million records, an error 500 appeared on the screen, and I noticed that the clickhouse database stopped responding. After 20 minutes, the database started working, but all the tables in it were empty.
It seems that the database from a large number of operations goes into reboot, from which it comes out completely empty.

Why is this happening?

**Comment by den-crane** [comment]
you should ask Yandex Cloud support.

Also check `system.detached_parts` table.
